cluster/gce/gci/configure-helper.sh

```diff
@@ -53,6 +53,9 @@ function main() {
   CONTAINERIZED_MOUNTER_HOME="${KUBE_HOME}/containerized_mounter"
   PV_RECYCLER_OVERRIDE_TEMPLATE="${KUBE_HOME}/kube-manifests/kubernetes/pv-recycler-template.yaml"
   DEFAULT_NETWORK_TEMPLATE="${KUBE_HOME}/kube-manifests/kubernetes/default-network.tmpl"
+  DEFAULT_PATH="/hack/runtime"
```

Comment:
"/hack/runtime" looks like a Dev machine folder, should we change this to match with other file folder?

Reply:
This env is to match the path defined in golang code line 45 in cmd/kube-controller-manager/app/mizarcontrollers.go.

homeSubPath = "/hack/runtime/"

Reply:
Ever think another idea: first download the template file into ${KUBE_HOME}/kube-manifests/kubernetes/<filename>, then create soft link /hack/runtime/<filename> to ${KUBE_HOME}/kube-manifests/kubernetes/<filename>

Reply:
In master branch, will use json struct object to replace json template file and finally fix the above concern.

Reply:
This kind of hard code should be avoided from source code.

Reply:
The parameter '--default-network-template-path' for kube-controller-manager is provided, so we can set default-network-template-path to any path which we want.

--default-network-template-path=/home/kubernetes/kube-manifests/kubernetes/default-network.tmpl

But no parameters of vpc template and subnet template are provided for mizar-arktos-network-controller transferred by kube-controller-manager, so currently /hack/runtime is temporarily used.

```
ubuntu@ip-172-31-14-204:~/go/src/arktos$ ls -al ./hack/runtime
total 40
drwxrwxr-x  2 ubuntu ubuntu  4096 Jan 27 22:30 .
drwxrwxr-x 14 ubuntu ubuntu 12288 Feb  2 02:05 ..
-rw-rw-r--  1 ubuntu ubuntu   154 Dec  7 19:58 default_flat_network.json
-rw-rw-r--  1 ubuntu ubuntu   197 Dec  7 19:58 default_mizar_network.json
-rw-rw-r--  1 ubuntu ubuntu   285 Jan 27 22:30 default_mizar_network_subnet_template.json
-rw-rw-r--  1 ubuntu ubuntu   230 Jan 27 22:30 default_mizar_network_vpc_template.json
-rw-rw-r--  1 ubuntu ubuntu   333 Dec  7 19:58 workload-controller-manager-clusterrole.yaml
-rw-rw-r--  1 ubuntu ubuntu   321 Dec  7 19:58 workload-controller-manager-clusterrolebinding.yaml
```

Reply:
If the following parameters are provided for kube-controller-manager, the situation will be better, but new codes have to be added into tenant-controller.  However, we decide to use json struct object to replace json template file to avoid using hard code "/hack/runtime".

--default-vpc-template-path
--default-template-template-path